---
bool object_found
bool obstacle_found
---
string message